packages/blockprotocol/core.d.ts

```diff
@@ -22,22 +22,79 @@ export type BlockMetadataRepository =
   | string;
 
 export type BlockMetadata = {
-  author?: string | null;
-  default?: JSONObject | null;
-  description?: string | null;
-  displayName?: string | null;
-  examples?: JSONObject[] | null;
-  externals?: Record<string, string> | null;
-  icon?: string | null;
-  image?: string | null;
-  license?: string | null;
-  name?: string | null;
-  protocol?: string | null;
+  /**
+   * The name of the author of the block
+   */
+  author: string;
+  /**
+   * The default object used to populate a block's schema
+   */
+  default?: {
+    [k: string]: unknown;
+  };
+  /**
+   * Describes what a block does
+   */
+  description: string;
+  /**
+   * The display name used for a block
```

Comment:
Can you add this to the list of SHOULDs in the spec please?

Reply:
Added here: https://github.com/blockprotocol/blockprotocol/pull/232/files#diff-729c25c2d4d8e3d3a0bc813c45567e791a83065ea1d749aa96166ef7f166bd46R48